IyiKuyoro/portfolio-react
# #170446561: Stop Animations on Demand

#### What does this PR do?

This PR adds the ability to toggle the animations feature on or off

#### Description of Task to be completed?

- Add handicap button
- Make handicap menu draggable
- Add tooltips
- Add menu list
- Add toggle animations button

#### How should this be manually tested?

- Launch the app and navigate to the home screen
- Using the accessibility menu, toggle the animations button off
- Ensure that the animations stop
- Also, ensure that one can still scroll over the projects

#### What are the relevant pivotal tracker stories?

#170446561

#### Any background context you want to add?

As an accessibility issue, some users may find the animations a bit annoying. This feature will ensure these users are considered.

#### Screenshots

<img width="431" alt="Screenshot 2019-12-28 at 21 39 19" src="https://user-images.githubusercontent.com/30635021/71549242-83cb0e00-29ba-11ea-9ddd-782d89f2915f.png">
